src/components/accessibility/skip_link/skip_link.stories.tsx

```diff
@@ -7,13 +7,22 @@
  */
 
 import type { Meta, StoryObj } from '@storybook/react';
-import { hideStorybookControls } from '../../../../.storybook/utils';
 
+import {
+  hideStorybookControls,
+  moveStorybookControlsToCategory,
+} from '../../../../.storybook/utils';
 import { EuiSkipLink, EuiSkipLinkProps } from './skip_link';
 
 const meta: Meta<EuiSkipLinkProps> = {
   title: 'EuiSkipLink',
   component: EuiSkipLink,
+  argTypes: {
+    ...moveStorybookControlsToCategory(
```

Comment:
[optional comment, feel free to go with your own preference] If we're not setting any other `argTypes`, I'm personally fine not making this a spread - totally up to you though!
```suggestion
  argTypes: moveStorybookControlsToCategory(
```